dot/config.go

```diff
@@ -120,6 +122,21 @@ func RPCServiceEnabled(cfg *Config) bool {
 	return cfg.RPC.Enabled
 }
 
+// RPCExternalServiceEnabled returns true if the rpc service accepts external connections
+func RPCExternalServiceEnabled(cfg *Config) bool {
+	return cfg.RPC.ExternalEnabled
+}
+
+// WSServiceEnabled returns true if the websocket service is enabled
+func WSServiceEnabled(cfg *Config) bool {
+	return cfg.RPC.WSEnabled
+}
+
+// WSExternalServiceEnabled returns true if the websocket service accepts external connections
+func WSExternalServiceEnabled(cfg *Config) bool {
+	return cfg.RPC.WSExternalEnabled
+}
+
```

Comment:
why is this needed? you should be able to directly the fields from the `Config` since they're exported

Reply:
@noot Added them there purely because I saw other config items exported in a similar way, will remove